build.sbt

```diff
@@ -158,15 +158,18 @@ val noBinaryCompatCheck = Set[String]("benchmark", "caliper", "generic", "spark"
 
 def previousVersion(subProj: String) =
   Some(subProj)
-    .filterNot(noBinaryCompatCheck.contains(_))
-    .map { s => "com.twitter" %% ("algebird-" + s) % "0.13.4" }
+    .filterNot(noBinaryCompatCheck.contains)
+    .map { s => "com.twitter" %% ("algebird-" + s) % "0.13.5" }
```

Comment:
with `mima < 0.4.0` some rules where missed and nwe ones were [introduced](https://users.scala-lang.org/t/announcing-mima-0-4-0/4799). `0.13.4` and `0.13.5` might not be binary compatible.